Makefile

```diff
@@ -92,44 +106,19 @@ prereq:
 	GOROOT=${GO_DIR} GOPATH=$(HOME)/app ${GO_BIN_PATH}/go install github.com/golang/protobuf/protoc-gen-go@v1.5.2
 
 protos:
-	@cd src; \
+	@cd svc-device-manager; \
 	GOROOT=${GO_DIR} GOPATH=$(HOME)/app PATH=$(PATH):$(HOME)/app/bin protoc --proto_path=proto \
```

Comment:
Maybe we can move building proto for Device Manager into build/buildProtoFiles.sh scrip? Same as other services.

Reply:
This is legacy proto, not required for integration. Technically it could get removed but we opted to keep legacy code, for now.